docs/configuration.md

```diff
@@ -767,3 +775,70 @@ sanityRobo = true
 ``` kotlin
 sanityRobo.set(true)
 ```
+
+### defaultTestTime
+Set default test time expressed in seconds, used for calculating shards. (default: 120.0s)
+
+=== "Groovy"
+``` groovy
```

Comment:
i think these need to be indented like the other ones in order for the syntax to work.
```suggestion
    ``` groovy
    defaultTestTime = 1.2
    ````
```